project/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -51,6 +51,9 @@ Fixes #
 - [ ] Update the tests
 - [ ] Update the documentation
 - [ ] Add an upgrade note
+- [ ] Give a valid reason for targetting this branch
```

Comment:
Not a big fan about a todo for issue templating fill. We already have comment for that.

Plus, if the contributor didn't care of this template from the start, he will not care of this task list either.